test/testinput/bufr_ncep_1bamua.yaml

```diff
@@ -67,101 +86,92 @@ observations:
           path: "*/BRITCSTC"
 
       globals:
-        - name: "MetaData/platformCommonName"
+        - name: "platformCommonName"
           type: string
           value: "AMSU-A"
 
-        - name: "MetaData/platformLongDescription"
+        - name: "platformLongDescription"
           type: string
           value: "MTYP 021-023 PROC AMSU-A 1B Tb"
 
-        - name: "MetaData/sensorCentralFrequency"
-          type: floatVector
-          value: [23.800, 31.400, 50.300, 52.800, 53.596, 54.400, 54.940, 55.500, 57.290,
-                  57.290, 57.290, 57.290, 57.290, 57.290, 89.000]
-
       variables:
+        # MetaData
         - name: "MetaData/satelliteIdentifier"
           source: variables/satelliteIdentifier
-          longName: "SATELLITE IDENTIFIER"
-          units: ""
+          longName: "Satellite Identifier (e.g. 4 for Metop-a)"
 
         - name: "MetaData/satelliteInstrument"
           source: variables/satelliteInstrument
-          longName: "SATELLITE INSTRUMENT"
-          units: ""
+          longName: "Satellite Instrument"
 
-        - name: "MetaData/channelNumber"
-          source: variables/channelNumber
-          longName: "CHANNEL NUMBER"
-          units: ""
+        - name: "MetaData/sensorChannelNumber"
+          source: variables/sensorChannelNumber
+          longName: "Sensor Channel Number"
 
         - name: "MetaData/latitude"
           source: variables/latitude
-          longName: "LATITUDE (COARSE ACCURACY)"
-          units: "degree (north)"
+          longName: "Latitude (coarse accuracy)"
```

Comment:
Remove (coarse accuracy) as it is not needed.

Reply:
@YoulongXia-NOAA 

The heightOfStation when applies to satellite data means the altitude of the satellite from the surface of the earth (~ 800 km for polar orbiting satellite; ~ 36,000 km for geostationary satellite)

Reply:
@emilyhcliu, can we use heightOfSatellite or altitudeOfSatellite? Here Station may confuse the users. What do you think?